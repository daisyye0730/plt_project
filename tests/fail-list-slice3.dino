int main() {
  List(int, 4) li;
  List(int, 1) subli;

  li = [1, 2, 3, 4];
  subli = li[2:2];
  
}