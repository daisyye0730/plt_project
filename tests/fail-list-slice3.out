Fatal error: exception Failure("beginning and end indices are the same. Use list indexing instead")
